diag_integral/diag_integral.F90

```diff
@@ -1522,7 +1527,40 @@ function vert_diag_integral (data, wt) result (data2)
 
 end function vert_diag_integral
 
-
+function ensemble_file_name(fname) result(updated_file_name)
+     character (len=mxch), intent(inout) :: fname
+     character (len=mxch) :: updated_file_name
+     integer :: ensemble_id_int
+     character(len=7) :: ensemble_suffix
+     character(len=2) :: ensemble_id_char 
+     integer :: i
+     !> Make sure the file name short enough ti handle adding the ensemble number
```

Comment:
minor typo: 'ti' should be 'to'

Reply:
I am the master of the typo.  It is fixed now.